pkg/altbn128/altbn128.go

```diff
@@ -9,10 +9,12 @@ import (
 	"github.com/keep-network/keep-core/pkg/internal/byteutils"
 )
 
+// G1Point represents G1 point
```

Comment:
We need to add dots at the ends of all these comments we're adding in this PR.

Reply:
Fixed in abb1a2151c4f605c7447e4817f710d80c01fe4b8.